browser/brave_news/brave_news_tab_helper.h

```diff
@@ -30,28 +30,36 @@ class BraveNewsTabHelper
   struct FeedDetails {
     GURL feed_url;
     std::string title;
+
+    // Indicates whether we've requested this feed, so we don't request it
+    // multiple times.
+    bool requested_feed;
```

Comment:
It'd be better to have default value so that we don't have garbage value.

```suggestion
    bool requested_feed = false;
```